.storybook/preview.tsx

```diff
@@ -165,9 +119,65 @@ const analyticsSettingsDecorator = (Story, context) => {
   return <Story {...context} />;
 };
 
-export const decorators = [
-  onDarkDecorator,
-  languageSettingDecorator,
-  analyticsSettingsDecorator,
-  themeSettingDecorator,
-];
+const preview: Preview = {
+  globalTypes: {
+    language: {
+      name: 'Language',
+      description: 'Internationalization language',
+      defaultValue: 'en',
+      toolbar: {
+        icon: 'globe',
+        items: [
+          { value: 'en', title: 'English' },
+          { value: 'es', title: 'Español' },
+        ],
+      },
+    },
+    analytics: {
+      name: 'Analytics',
+      description: 'Analytics settings',
+      defaultValue: 'off',
+      toolbar: {
+        icon: 'graphline',
+        items: [
+          { value: 'on', left: 'Analytics', title: 'Log to Actions' },
+          { value: 'off', left: 'Analytics', title: 'Off' },
+        ],
+      },
+    },
+    theme: {
+      name: 'Theme',
+      description: 'Current theme',
+      defaultValue: 'core',
+      toolbar: {
+        icon: 'paintbrush',
+        items: Object.keys(themes).map((key) => ({
+          value: key,
+          title: `${themes[key].displayName} theme`,
+        })),
+      },
+    },
+  },
+  parameters: {
+    actions: { argTypesRegex: '^on[A-Z].*' },
+    backgrounds: { disable: true },
+    controls: {
+      expanded: true,
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/,
+      },
+    },
+    viewport: {
+      viewports: breakpointViewportSizes,
```

Comment:
When testing, you'll notice a (L) or a (P) next to the selected viewport. This is to denote "landscape" or "portrait" orientation. I can't find a way to change this indicator - it'd be great if I could have it say the thing it's actually doing instead of abbreviating.